assets/differentialabundance_report.Rmd

```diff
@@ -896,6 +896,7 @@ for (i in 1:nrow(contrasts)){
             contrast_de <- round_dataframe_columns(contrast_de, digits=params$report_round_digits)
             print( htmltools::tagList(datatable(contrast_de, caption = paste('Differential genes', dir, 'in', contrast_descriptions[i], " (check", differential_files[[i]], "for more detail)"), rownames = FALSE) ))
 
+        if ("Gene biotype" %in% colnames(contrast_de)) {
```

Comment:
Sorry, didn't mean for you to completely remove indenting, it just didn't seem consistent when I looked yesterday (apologies if it was just me having a bad day) - can you indent the lines inside?

I'll approve in the meantime so you're not blocked when that's done.